enhancements/teps/0001-option-to-disable-default-injection-of-serviceaccount-rbac.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Option to disable dafault creation of service account and rbac resources
+authors:
+  - "@savitaashture"
+creation-date: 2021-09-19
+last-updated: 2021-09-19
+status: implementable
+---
+
+# TEP-0001: Option to disable dafault creation of service account  and rbac resources
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+  - [Usage examples](#usage-examples)
+- [Design Details](#design-details)
+  - [Disable creation of RBAC resources at cluster level](#disable-creation-of-rbac-resources-at-cluster-level)
+  - [Disable creation of RBAC resources at namespace level](#disable-creation-of-rbac-resources-at-namespace-level)
+  - [Scenarios](#scenarios)
+- [Impact](#impact)
+<!-- /toc -->
+
+## Summary
+
+The proposal helps cluster admin to disable auto creation of RBAC resources
+(ServiceAccount, RoleBinding, SCCRoleBinding, CABundlesConfigMap and add Subjects to openshift-pipelines-clusterinterceptors ClusterRoleBinding).
+Related Stories and Epics are
+
+https://issues.redhat.com/browse/SRVKP-1578
+
+https://issues.redhat.com/browse/SRVKP-1670
+
+https://issues.redhat.com/browse/SRVKP-1649
+
+## Motivation
+
+OpenShift Pipelines operator will create a RBAC resources ((`pipeline`)ServiceAccount, RoleBinding, SCCRoleBinding, CABundlesConfigMap and Subjects to openshift-pipelines-clusterinterceptors) on all namespace when installed.
+The pipelines-scc-rolebinding has RunAsAny among other things.
+This can be seen as a security issue as some customers have reported. It would be interesting if a cluster-admin could disable this feature on basis of requirement.
+
+### Goals
+
+Provide way to cluster admin to disable auto creation of rbac resources at cluster as well as namespace level. 
+
+### Non-Goals
+
+## Proposal
+
+Installation of OpenShift Pipelines operator by default create RBAC resources on all namespaces.
+cluster admin will get permission to disable RBAC resource creation at cluster level using `TektonConfig CR` and at namespace level using `label`. 
+   
+### User Stories
+Some customers have reported that SCCRolebinding `pipelines-scc-rolebinding` can be seen as a security issue which has **RunAsAny** among other things.
```

Comment:
A user story should be something more like `As a cluster-admin, I want to be able to …`